### PR TITLE
Xenoglossy

### PR DESCRIPTION
**Pull request by Elijahrane**
# Description
Adds xenoglossy to the power pool. I originally planned it for cataloguer but it looks like that doesn't exist anymore.

# Changelog

:cl: Rane
- add: Added Xenoglossy to the psionic power pool.

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]
Hilariously enough this was on my list of powers to create. Thanks!

**Comment by stellar-novas** [comment]
Well, we could bring Cataloguer back, it would give the role a purpose

**Comment by VMSolidus** [comment]
And here's the first ever "Zero C#" Psionic Power. That's incredible to witness.

**Comment by VMSolidus** [comment]
> I originally planned it for cataloguer but it looks like that doesn't exist anymore.

Then let's add it back after this is merged.

File: Resources/Prototypes/Psionics/psionics.yml [hunk 1]
```diff
@@ -109,4 +109,13 @@
...
+  dampeningModifier: 0.5
+
+- type: psionicPower
+  id: XenoglossyPower
+  name: Xenoglossy
+  description: xenoglossy-power-description
+  components:
+    - type: UniversalLanguageSpeaker
+  initializationFeedback: xenoglossy-power-initialization-feedback
+  metapsionicFeedback: psionic-language-power-feedback #Reuse for telepathy, clairaudience, etc
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Psionics/psionics.yml:121]:
```suggestion
  metapsionicFeedback: psionic-language-power-feedback # Reuse for telepathy, clairaudience, etc

```

**Review by DangerRevolution — APPROVED** [review]
works in game

**Comment by AeraAuling** [comment]
> > I originally planned it for cataloguer but it looks like that doesn't exist anymore.
> 
> Then let's add it back after this is merged.

librarian is just cataloguer renamed and with epi access/monitor removed. early on in the rebase, there were even some maps that hadn't had epi doors removed from their librarian rooms which made for some funny moments where you just couldn't access your own office. otherwise, the two are basically identical.

**Pull request merged**